metaphor/tableau/extractor.py

```diff
@@ -57,6 +58,7 @@ async def extract(self, config: TableauRunConfig) -> List[MetadataChangeEvent]:
         logger.info("Fetching metadata from Tableau")
 
         self._snowflake_account = config.snowflake_account
+        self._preview_image = not config.disable_preview_image
```

Comment:
nit. Why introduce a new name?
```suggestion
        self._disable_preview_image = not config.disable_preview_image
```

Reply:
Reverse logic is just a bit harder to read :)

```
if not self._disable_preview_image:
```

vs

```
if self._preview_image:
```

Reply:
Actually now it's only used in one place, dropping the class variable.